src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/RequireSsoPolicyRequirement.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Bit.Core.AdminConsole.Enums;
+using Bit.Core.AdminConsole.Models.Data.Organizations.Policies;
+using Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+using Bit.Core.Enums;
+using Bit.Core.Settings;
+
+/// <summary>
+/// Policy requirements for the Require SSO policy.
+/// </summary>
+public class RequireSsoPolicyRequirement : IPolicyRequirement
+{
+    private readonly IEnumerable<PolicyDetails> _policyDetails;
+
+    public RequireSsoPolicyRequirement(IEnumerable<PolicyDetails> policyDetails)
+    {
+        _policyDetails = policyDetails;
+    }
+
+    /// <summary>
+    /// Indicates whether the user can use passkey login.
+    /// Policy is enforced for users with status >= Accepted.
+    /// </summary>
+    public bool CanUsePasskeyLogin => _policyDetails.Any(p =>
+        p.OrganizationUserStatus >= OrganizationUserStatusType.Accepted);
```

Comment:
Maybe this is why the tests are passing... this bool is also flipped. You can only use passkey login if RequireSSO _doesn't_ apply to you, but this returns true if it _does_ apply. So it's flipped here and then flipped back in the caller.

I think this should be `!_policyDetails.Any`.